contentcuration/contentcuration/assets/shared/vuex/baseStore.js

```diff
@@ -0,0 +1,26 @@
+import Vue from 'vue';
+import Vuex from 'vuex';
+import session from './session';
+
+Vue.use(Vuex);
+
+export default function storeFactory({state, actions, getters, mutations, modules, plugins} = {
+  state: {},
+  actions: {},
+  getters: {},
+  mutations: {},
+  modules: {},
+  plugins: [],
+}) {
+  return new Vuex.Store({
+    state,
+    actions,
+    getters,
+    mutations,
+    plugins,
+    modules: {
+      session,
```

Comment:
Adds any default modules here, only session is shared so far - this is done to completely replace our current global State object.